Vagrantfile

```diff
@@ -27,8 +26,7 @@ $bootstrap = <<BOOTSTRAP
   sudo chown -R vagrant /var/lib/
   sudo chown -R mysql /var/log/mysql
   sudo chown -R mysql /var/log/mysql
-  sudo apt-fast -y install emacs24-nox
-  sudo apt-fast -y install libmagickwand-dev
+  sudo apt-fast -y install emacs24-nox findutils realpath libmagickwand-dev
```

Comment:
On 09/01/2016 10:46 PM, Jon Kiparsky wrote:

> emacs is, strictly speaking, not required for our build. Can we come up
> with a way to make it optional, so that there's less build time for
> people who don't want it on their vagrant box?

Yes....  But not right now.  Pulling it for the time being, and waiting 
until I start looking at some of the other improvements I would want to 
do on the Vagrantfile....  Like having an external port setting so two 
box do not fight for 8282.

Hunter